save-cosv/src/main/kotlin/com/saveourtool/save/cosv/controllers/RawCosvFileController.kt

```diff
@@ -387,6 +374,34 @@ class RawCosvFileController(
                 }
         }
 
+    /**
+     * @param organizationName
+     * @param authentication
+     * @return [StringResponse]
+     */
+    @RequiresAuthorizationSourceHeader
+    @DeleteMapping("/delete-all-duplicated-files")
+    fun deleteAllDuplicatedFiles(
+        @PathVariable organizationName: String,
+        authentication: Authentication,
+    ): Mono<StringResponse> = hasPermission(authentication, organizationName, Permission.DELETE, "delete")
+        .flatMap {
+            rawCosvFileStorage.listByOrganizationAndUser(organizationName, authentication.name)
+                .map { files ->
+                    files.filter { it.isDuplicate() }.map { it.requiredId() }
+                }
+                .flatMap { ids ->
+                    rawCosvFileStorage.deleteAllByIds(ids)
```

Comment:
you list all files, takes ids for them and then delete by ids by searching these keys.
can you please add a method `deleteAll(keys: Collection<K>)` instead of `deleteAllByIds`?

Reply:
Done